### ISSUE TITLE
Feature request: save list of installed games and their update version

### ISSUE THREAD

**Issue by ShidaPenns** [XorTroll/Goldleaf#249]
Would this be possible? I have a 400 GB micro SD so I have a lot of installed games. It would make it easier to find the latest update for them if I could open a text file with this info.

**Comment by lookye** [comment]
Do you mean something like this:
Open Goldleaf -> Manage console contents -> Choose your SD card -> choose a title or the content (dlc/update)

If you want the info, if there is a newer update for the titles you installed. No, this is probably not possible. You need to look for it on your own. There are enough places to look for in the internet.

**Comment by ShidaPenns** [comment]
No, I don't want it to look and see if there's a newer update, I just mean saving to a .txt or something a list of the installed games and their *currently installed* update version, and I check if there's a newer update myself. If that's not possible, then fair enough.

**Comment by satelliteseeker** [comment]
Preferably in JSON format for machine readability 🤔

**Comment by XorTroll** [comment]
So like get update info from Ninty's servers?

**Comment by ShidaPenns** [comment]
I mean, if that's possible and not a bad idea (i.e. lead to a quick ban), sure. But my plan was just, look at what's installed on the Switch and record that. Making it easier to find what the latest update is, myself, on my PC.
